OvaryModelGenerator/PelvisModelGenerator.h

```diff
@@ -4,23 +4,24 @@
 #include <iostream>
 
 #include "vtkPolyData.h"
+#include "vtkSmartPointer.h"
 
 class PelvisModelGenerator{
 
-    public:
-        float intercristalDistance;
-        float pelvisScaleFactor;
-        vtkPolyData* pelvisOutputPolydata;
-
     public:
         PelvisModelGenerator();
         ~PelvisModelGenerator();
-        PelvisModelGenerator(const PelvisModelGenerator &other);
+        PelvisModelGenerator(const PelvisModelGenerator &other) = delete;
+        PelvisModelGenerator & operator=(const PelvisModelGenerator &other) = delete;
 
         void SetIntercristalDistancemm(float intercristalDistance); //sets intercristal distance of pelvis and scale factor
         void Generate();
         vtkPolyData* GetOutput();
 
+    private:
```

Comment:
Internal data should be kept private and add public accessor such as GetXXXData(); SetXXXData(); for indirect public access.